planner/core/logical_plan_test.go

```diff
@@ -991,13 +991,6 @@ func (s *testPlanSuite) TestColumnPruning(c *C) {
 				3: {"a"},
 			},
 		},
-		//issue 7833
-		{
```

Comment:
So, why this is removed?

Reply:
"drop view if exists v" is changed to `AlterTableStmt`, which is built as a core.DDL...

Reply:
It seems my fault...this test case has been remove now.